### PR TITLE
fix(anthos): fix tracking

### PR DESCRIPTION
**Pull request by ghyenne**
ref: MANAGER-8622
Signed-off-by: Guillaume Hyenne <guillaume.hyenne@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos-m4` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-8622 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
Fix tracking issue on resiliation

### CODE REVIEW

File: packages/manager/modules/anthos/src/dashboard/routing.js [hunk 1]
```diff
@@ -112,7 +112,13 @@ export default /* @ngInject */ ($stateProvider) => {
...
 
       serviceInfo: /* @ngInject */ (serviceName, AnthosTenantsService) =>
-        AnthosTenantsService.getServiceInfo(serviceName),
+        AnthosTenantsService.getServiceInfo(serviceName).then((data) => {
+          const serviceInfo = {
+            ...data,
+            serviceType: SERVICE_TYPE,
+          };
+          return serviceInfo;
+        }),
```
Comment (y4nnL) [inline packages/manager/modules/anthos/src/dashboard/routing.js:121]: Just a suggestion (not a blocker)
```suggestion
        AnthosTenantsService.getServiceInfo(serviceName).then((data) => ({
            ...data,
            serviceType: SERVICE_TYPE,
        }),
````

File: packages/manager/modules/anthos/src/dashboard/constants.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 export const TRACKING_PREFIX = 'hpc::anthos::project';
+export const SERVICE_TYPE = 'DEDICATED_ANTHOS';
```
Comment (y4nnL) [inline packages/manager/modules/anthos/src/dashboard/constants.js:2]: question : are you sure the requested `serviceType` should not be `dedicated-anthos` ?

**Review by y4nnL — CHANGES_REQUESTED** [review]
1 blocker

File: packages/manager/modules/anthos/src/dashboard/routing.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { ANTHOS_TENANT_ALERTER } from '../anthos.constants';
```
Comment (y4nnL) [inline packages/manager/modules/anthos/src/dashboard/routing.js:1]: **Blocker** : the changes should me made at the top of the service, in
`packages/manager/modules/anthos/src/anthos-tenants.service.js`
Within the function `getServiceInfo` :+1:

**Review by y4nnL — CHANGES_REQUESTED** [review]

File: packages/manager/modules/anthos/src/dashboard/constants.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/modules/anthos/src/dashboard/constants.js]: It is not possible to put the lower case version directly.
I think it's linked to kebabCase.
```
dedicated::account::billing::autorenew::${kebabCase(
  $state.transition.params().serviceType,
)}::delete
```

**Review by Jisay — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**